File: Salon/src/main/java/com/example/salon/dto/BookingResponse.java
package com.example.salon.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BookingResponse {
    private Long id;
    private ServiceResponse service;
    private LocalDateTime startTime;
    private LocalDateTime endTime;
    private String customerName;
    private String customerPhone;
    private String status;
    private LocalDateTime createdAt;
}

File: BlazorSalonApp/BlazorSalonApp/Dto/BookingResponse.cs
namespace BlazorSalonApp.Dto;


public class BookingResponse
{
    public long Id { get; set; }
    public ServiceResponse Service { get; set; } = new();
    public DateTime StartTime { get; set; }
    public DateTime EndTime { get; set; }
    public string CustomerName { get; set; } = string.Empty;
    public string CustomerPhone { get; set; } = string.Empty;
    public string Status { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
}